unit_tests/HttpClient.d

```diff
@@ -0,0 +1,326 @@
+// HTTP utils based on https://github.com/adamdruppe/arsd/blob/master/http.d
+// copyright Adam D. Ruppe
+module HttpClient;
+
+import std.socket;
+import std.string;
+import std.conv;
+static import std.algorithm;
+static import std.uri;
+
+ubyte[] getBinary(string url, string[string] cookies = null) {
+	auto hr = httpRequest("GET", url, null, cookies);
+	if(hr.code != 200)
```

Comment:
Looks like it doesn't handle redirects, but I guess we can live without that until it's actually needed.